plugins/amongus/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Mumble Developers. All rights reserved.
+# Copyright 2009-2021 The Mumble Developers. All rights reserved.
```

Comment:
Something is not correct here. The script must be bugged
(This file did definitely not exist back in 2009)